src/services/measure.service.js

```diff
@@ -42,4 +45,72 @@ const remove = async args => {
   return baseRemove(args, 'Measure');
 };
 
-module.exports = { create, searchById, remove, update };
+/**
+ * takes a measureReport and a set of required data with which to calculate the measure and
+ * creates new documents for the measureReport and requirements in the appropriate collections
+ * @param {*} args the args object passed in by the user
+ * @param {*} req the request object passed in by the user
+ * @returns a transaction-response bundle
+ */
+const submitData = async (args, { req }) => {
+  if (req.body.resourceType !== 'Parameters') {
+    throw new ServerError(null, {
+      statusCode: 400,
+      issue: [
+        {
+          severity: 'error',
+          code: 'BadRequest',
+          details: {
+            text: `Expected 'resourceType: Parameters'. Received 'type: ${req.body.resourceType}'.`
+          }
+        }
+      ]
+    });
+  }
+  if (!req.body.parameter) {
+    throw new ServerError(null, {
+      statusCode: 400,
+      issue: [
+        {
+          severity: 'error',
+          code: 'BadRequest',
+          details: {
+            text: `Unreadable or empty entity for attribute 'parameter'. Received: ${req.body.parameter}`
+          }
+        }
+      ]
+    });
+  }
+
+  const tb = new TransactionBundle();
+  const parameters = req.body.parameter;
+
+  let containsMeasureReport = false;
+
+  parameters.forEach(param => {
+    //TODOMAYBE: add functionality for if resource is itself a bundle
```

Comment:
I agree that functionality should be added if the resource is a bundle, and it looks like `cqf-ruler` accounts for this. Should this be added as a separate task?